frontend/simple/utils/distribution/mincome-default.js

```diff
@@ -49,20 +50,27 @@ function transferValue (incomes, indexAmount, transferAmount) {
   })
 }
 
-function incomeKeys (a, b) {
-  return a.name > b.name
+function incomeKeys (a: Object, b: Object): number {
```

Comment:
It's a good idea to validate `a` and `b` keys too. Something like:

```
function incomeKeys (a: { name: number  }, b: { name: number  }: number {
```

Reply:
nice, thx